torch/testing/_internal/common_distributed.py

```diff
@@ -249,7 +249,7 @@ def create_tcp_store(addr="localhost", world_size=1, is_master=True, timeout=tim
         return c10d.TCPStore(addr, port, world_size, is_master, wait_for_workers=wait_for_workers)
 
 
-TIMEOUT_DEFAULT = 100
+TIMEOUT_DEFAULT = 300
```

Comment:
```suggestion
TIMEOUT_DEFAULT = int(os.getenv('DISTRIBUTED_TESTS_DEFAULT_TIMEOUT', '100'))
```

Reply:
I'm not opposed to that, as it is an improvement over having to maintain a patch.
However: Shouldn't unit tests always succeed and not cause false positive failures, even when they are run in a different (but non-arcane) environment/CI?